logicrunner/executionbroker.go

```diff
@@ -304,7 +308,7 @@ func (q *ExecutionBroker) startProcessors(ctx context.Context) {
 
 func (q *ExecutionBroker) startProcessorCount(ctx context.Context, state *processorState) int {
 	queueLength := state.queue.Length()
-	workerAvailable := state.limiter.Available(ctx)
+	workerAvailable := state.workersLimit - state.workers
```

Comment:
I would like this and state.workersLimit == state.workers to be helpers on state, imho it would look better

Reply:
fixed